components/ui/AppCta.vue

```diff
@@ -37,5 +37,26 @@ export default class extends Vue {
   & :first-child {
     margin-right: $layout-02;
   }
+  svg {
```

Comment:
Name the icon explicitly as a BEM Element of `app-cta`. Something like `app-cta__icon` should be enough.